lang/python/python/Makefile

```diff
@@ -263,6 +264,9 @@ define PyPackage/python-base/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON_VERSION).so* $(1)/usr/lib/
 endef
 
+PyPackage/python-light/install?=:
+PyPackage/python/install?=:
```

Comment:
A very small nitpick: `?=` sets a variable if it hasn't already been set (which is why it works for `PyBasePackage`).

Can you change these to `:=`, so:
```
PyPackage/python-light/install:=:
PyPackage/python/install:=:
```

Reply:
Changed..